### PR TITLE
Slack

### PR DESCRIPTION
**Pull request by wence-**
Ahead of this afternoon's discussion.

### CODE REVIEW

**Comment by dorugeber** [comment]
This looks like 2 (or more) unrelated sets of changes?

**Comment by wence-** [comment]
Oh, yes.

**Comment by wence-** [comment]
fixed.

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ notifications:
     on_success: change
     on_failure: always
     template: "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message} | %{build_url}"
+  slack: firedrakeproject:8zAJ7fho63HNWPbKMrifGmZY
```
Comment (wence-) [inline .travis.yml]: Should use https://docs.travis-ci.com/user/encryption-keys/ to not share this secret info.

Comment (wence-) [inline .travis.yml]: done.

**Pull request merged**